libs/blocks/global-navigation/utilities/menu/menu.css

```diff
@@ -197,4 +201,51 @@
   .feds-promo--dark a:hover {
     text-decoration: underline;
   }
+
+
+  .feds-links-strip-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    border-top: solid 1px #f3f3f3;
+  }
+
+  .feds-links-strip {
+    width: 100%;
+    padding: 20px 0;
+    max-width: var(--feds-maxWidth-nav);
+    display: flex;
+    justify-content: flex-start;
```

Comment:
This should happen by default, it probably doesn't need to be declared explicitly.

Reply:
True, removed it